buildbot_nix/__init__.py

```diff
@@ -172,6 +172,11 @@ def run(self) -> Generator[Any, object, Any]:
         # if the command passes extract the list of stages
         result = cmd.results()
         if result == util.SUCCESS:
+            log.info(self.observer.getStderr())
+            self.addHTMLLog(
+                "Evaluation Warnings", f"<pre>{self.observer.getStderr()}</pre>"
```

Comment:
Do you know if the string interpolation gets properly HTML-escaped?

Reply:
Uh, definitely not, good catch! JavaScript injection attacks disaprove :)

Reply:
I think the python stdlib has some helpers for that. Otherwise I am sure that buildbot has it somewhere.

Reply:
Indeed, but this was more of a prototype to showcase what can be done now anyway, but i don't think it's worth merging as it is, personally.